pkg/operator/starter.go

```diff
@@ -472,7 +473,8 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			os.Getenv("OPERATOR_IMAGE"),
 			featureGateAccessor,
 			backupVar,
-			configBackupInformer)
+			configBackupInformer,
+			etcdPodsInformer)
```

Comment:
```suggestion
			kubeInformersForNamespaces)
```

let's stick with the other core informers and pass this one down

Reply:
So i tried this in the beginning but it breaks the test 

Using this approach, the informer is being utilized whenever thr `clientSet` is used

This is out-of-the-box by the fabulous lib-go 

I can change it still, but i dont know how to fake informers in the test

Reply:
there are setups for all kinds of informer mocks in the tests

https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/etcdcertsigner/etcdcertsignercontroller_test.go#L556-L562

that's nothing new to this codebase

Reply:
Thxx 🙏🏽🙏🏽

I searched in membercontrollers and backuControllers not this one

Will do it today 🙏🏽🙏🏽